### PR TITLE
Dilpreet | Test | User Panel Sanity Issues

### PR DESCRIPTION
**Pull request by dilpreetsinghofficial**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/shared/header/header.component.html [hunk 1]
```diff
@@ -1,805 +1,773 @@
...
-                    [apiMenuItems]="apiMenuItems"
-                    [isGoToBranch]="isGoToBranch"
+            <div class="d-flex justify-content-between align-items-center h-100">
+                <button
+                    *ngIf="companyList?.length && isSubscriptionPage"
+                    class="min160 company-button"
+                    color="primary"
+                    mat-stroked-button
+                    (click)="backToCompany()"
+                    placement="bottom"
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/shared/header/header.component.html]: matTooltipPosition="below"

File: apps/web-giddh/src/app/shared/header/header.component.html [hunk 2]
```diff
@@ -1,805 +1,773 @@
...
+            subscribedPlan?.status !== 'expired' &&
+            remainingSubscriptionDays !== false &&
+            remainingSubscriptionDays <= 7
+        "
+        [ngClass]="{ 'menu_open': isSidebarExpanded && !isSubscriptionPage }"
+    >
+        <div class="white-row plan-issue">
+            <p>
+                <span class="icon-traingle-info"></span> {{ getSubscriptionEndNote() }}
+                <a href="javascript:void(0);" class="click-here" aria-label="go to plan" (click)="goToSelectPlan()">{{
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/shared/header/header.component.html:549]: javascript:;

File: apps/web-giddh/src/app/shared/header/header.component.html [hunk 3]
```diff
@@ -1,805 +1,773 @@
...
-            </div>
+    <div
+        class="plan-wrapper"
+        *ngIf="subscribedPlan !== undefined && subscribedPlan?.status === 'expired'"
+        [ngClass]="{ 'menu_open': isSidebarExpanded && !isSubscriptionPage }"
+    >
+        <div class="white-row plan-issue">
+            <p>
+                <span class="icon-traingle-info"></span> {{ getSubscriptionEndedNote() }}
+                <a href="javascript:void(0);" class="click-here" aria-label="get select " (click)="goToSelectPlan()">{{
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/shared/header/header.component.html:564]: javascript:;

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts [hunk 4]
```diff
@@ -484,6 +484,16 @@ export class BuyPlanComponent implements OnInit, OnDestroy {
         this.componentStore.activeCompany$.pipe(takeUntil(this.destroyed$)).subscribe(response => {
             if (response) {
                 this.activeCompany = response;
+                if (this.activeCompany) {
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts]: If you check response and after that assign response to activeCompany and then check again activeCompany 

I think no use check twice

File: apps/web-giddh/src/app/subscription/change-billing/change-billing.component.html [hunk 5]
```diff
@@ -69,20 +70,27 @@
...
-                                    autocomplete="off"
-                                    (keypress)="intlClass?.onlyPhoneNumber($event); validateMobileField()"
-                                    id="init-billing"
+                                    (keypress)="
+                                        intlClass?.onlyPhoneNumber($event);
+                                        changeBillingForm.controls['mobileNumber']?.markAsDirty()
+                                    "
+                                    [id]="'init-contact'"
                                     [placeholder]="commonLocaleData?.app_mobile_number"
-                                    formControlName="mobileNumber"
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/subscription/change-billing/change-billing.component.html:79]: check delete by mistake or intentionally

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

**Review by dvCodeWorld — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**